doc/ref_arch/kubernetes/chapters/chapter05.md

```diff
@@ -164,8 +164,8 @@ Sometimes container isolation is compared directly with VM based isolation, with
 - Containers: isolation by SW based mechanisms available in OS, Docker and Kubernetes. A container workload is just a set of Linux processes. It is _possible_ to configure SW based _additional isolation_ for container workloads, for example by kernel namespaces.
 
 
-Thus the primary isolation mechanism in Kubernetes environment should be VM or physical machine based isolation. This means: multiple container applications should not be deployed together in the same Kubernetes cluster - unless those have been planned and verified to co-exist. In such cases, the default is to allocate one namespace per Containerised Network Function (CNF).
+Thus the primary isolation mechanism in Kubernetes environment should be VM or physical machine based isolation. This means: multiple container applications should not be deployed together in the same Kubernetes cluster - unless those have been planned and verified to co-exist. In such cases, the default is to allocate one namespace per Cloud Native Network Function (CNF).
 
 ### Container Isolation in Kubernetes Cluster
 #### Namespaces  
-Kubernetes namespaces should be used to provide resource isolation within a Kubernetes cluster. They should not be used to isolate different steps in the deployment process like Development, Production, or Testing. The most reliable separation is achieved by deploying sensitive workloads into dedicated clusters. 
+Kubernetes namespaces should be used to provide resource isolation within a Kubernetes cluster. They should not be used to isolate different steps in the deployment process like Development, Production, or Testing. The most reliable separation is achieved by deploying sensitive workloads into dedicated clusters.
```

Comment:
Are "Development, Production, or Testing" steps in deployment process or they are different stages in the application development process.

Reply:
Not sure - note, I didn't actually make any changes here so would prefer to handle this in a separate issue/PR please.